### ISSUE TITLE
Suggestion: refer to rclone

### ISSUE THREAD

**Issue by jehon** [pcloudcom/console-client#187]
This project seem not maintained anymore.

But rclone is a well maintained and documented project that can handle pcloud easily. Why not using that project ?

https://rclone.org/pcloud/

... and asking pcloudcom to maintain the rclone project instead? ;-)

**Comment by alensiljak** [comment]
Pcloud does not seem to be maintaining much on GitHub in the recent years. And rclone does not do delta transfers.

**Comment by securiteinfo** [comment]
rclone does not support crypt feature of pcloud client

**Comment by alensiljak** [comment]
> rclone does not support crypt feature of pcloud client

True. There are also alternatives to that. Like Cryptomator or others.
